feature-libs/cart/added-to-cart-toast/core/models/added-to-cart-toast.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Images } from 'projects/core/src/model';
+
+export interface CartToastItem {
+  productName: string;
+  quantity: number;
+  unitPrice: string;
+  baseClass: string;
+  images?: Images;
+  timeoutRef?: ReturnType<typeof setTimeout>;
+  scrollUnlistener?: () => void;
```

Comment:
What is the purpose of this?

Reply:
This is for positioning the toast on scroll. I need to keep a reference for the unlistener when there is more than 1 active toast so that doesn't interfere with others.